scripts/package-distribution.ps1

```diff
@@ -6,6 +6,16 @@
 
 #>
 
+param (
+    [Parameter(Mandatory = $false)]
+    [switch]$Time64Distro
+)
+
+$Time64 = $false
```

Comment:
I think $Time64Distro is already boolean. why do you need to add one more variable?